### DISCUSSION TITLE
How to show unpublished entries on the frontend

### DISCUSSION THREAD

**Discussion by mikemartin** [statamic/cms#5559] [Q&A]
Very often I'll create a collection (for example: Features) where some entries have a full marketing page for that feature but the other entries are just stubs of content and shouldn't have a page at all. In this case, I've found that it's easiest to unpublish those entries to ensure their pages don't appear publicly or in the sitemap.xml.

When using the entries fieldtype, it's impossible to display those unpublished entries and it sounds like this is because unpublished entries are not augmented.

Are there any plans to fix this issue or make it easier to show entries that are unpublished on the frontend?

**Comment by duncanmcclean** [comment]
The issue will get fixed eventually but there's lots of other issues too 😅

**Comment by jasonvarga** [comment]
You already have https://github.com/statamic/cms/issues/3918 open. 😄
